### PR TITLE
Add an e2e test for RedisQuota Adapter

### PR DESCRIPTION
**Pull request by gargnupur**

### CODE REVIEW

File: mixer/adapter/redisquota/redisquota.go [hunk 1]
```diff
@@ -237,6 +241,8 @@ func (b *builder) Build(context context.Context, env adapter.Env) (adapter.Handl
 		dimensionHash: dimensionHash,
 	}
 
+	env.Logger().Infof("Redis Quota Adapter Built")
```
Comment (geeknoid) [inline mixer/adapter/redisquota/redisquota.go]: We don't really need this logging, Mixer is already handling this.

Comment (gargnupur) [reply mixer/adapter/redisquota/redisquota.go]: Thanks.. Removed

File: tests/e2e/tests/mixer/mixer_test.go [hunk 2]
```diff
@@ -1002,6 +995,77 @@ func TestMetricsAndRateLimitAndRulesAndBookinfo(t *testing.T) {
...
+	}
+	defer func() {
+		if err := util.HelmDelete(redisInstallName); err != nil {
+			t.Logf("Could not delete %s: %v", redisInstallName, err)
+		}
+	}()
+
+	allowRuleSync()
+
+	log.Info("Setting up port forwarding for redis server")
```
[the remark this answers is not in the document]
Comment (gargnupur) [reply tests/e2e/tests/mixer/mixer_test.go]: I was initially testing using localhost:6379 but that didn't really work.. thanks for catching.. removed this

**Comment by gargnupur** [comment]
Closing this pull request and sending a new one..

**Pull request closed without merging**